libmamba/src/api/install.cpp

```diff
@@ -63,12 +63,8 @@ namespace mamba
                           << " packages: " << join(", ", deps) << termcolor::reset;
         LOG_INFO << "Calling: " << join(" ", install_args);
 
-        auto [_, ec] = reproc::run(wrapped_command, options);
-
-        if (ec)
-        {
-            throw std::runtime_error(ec.message());
-        }
+        auto [status, ec] = reproc::run(wrapped_command, options);
+        assert_reproc_success(reproc::options::clone(options), status, ec);
```

Comment:
Looks all good to me. Only thing I am wondering about is this `clone` here. Is it not possible to use the options by reference or const reference?